client/src/pages/Login.jsx

```diff
@@ -1,141 +1,79 @@
 import { useMutation } from "@apollo/client";
-import React, { useState } from "react";
+import React from "react";
 import { Link } from "react-router-dom";
 import { LOGIN } from "../utils/mutations";
 import Auth from "../utils/auth";
-import {
-  Button,
-  Sheet,
-  Typography,
-  FormControl,
-  FormLabel,
-  Input,
-} from "@mui/joy";
+import { Button, Typography, Input, Card, FormHelperText } from "@mui/joy";
+import { useForm } from "react-hook-form";
+import { useSchemas } from "../hooks/useSchemas";
+import { yupResolver } from "@hookform/resolvers/yup";
 
-const Login = () => {
-  const [formData, setFormData] = useState({
-    email: "",
-    password: "",
-  });
-
-  const [formErrors, setFormErrors] = useState({
-    email: "",
-    password: "",
-  });
-
-  const handleChange = (event) => {
-    const { name, value } = event.target;
-    setFormData((prevData) => ({ ...prevData, [name]: value }));
-  };
+const styles = {
```

Comment:
Just a matter of preference, but this seems like it could live outside this module. Maybe in an adjacent style-dedicated file?

Reply:
Agreed. At some point I'd like to pass a theme to the entire front-end. Styling is what I have focused on the least so far in this project but I definitely think having larger bits of styling code in a separate file is where I will go with it.